cabal.project

```diff
@@ -10,3 +10,18 @@ repository cardano-haskell-packages
     d4a35cd3121aa00d18544bb0ac01c3e1691d618f462c46129271bccf39f7e8ee
 
 packages: .
+
+allow-newer:
```

Comment:
Why? We fix deps and at same time hack them? Should not that be on Atlas side then?